src/pages/partners/affiliate-ib/_deriv-ib-programme.tsx

```diff
@@ -250,6 +281,17 @@ const DerivIBProgramme = () => {
                         </>
                     )}
 
+                    {activeTab === 'cTrader' && (
+                        <>
+                            <Header as="h2" mb="3rem" type="page-title" align="center">
```

Comment:
duplication of h2 - bad approach to keep semantic of the page
better to refactor on just 'div' because h2 for header using as default